src/main/java/com/epam/emotionalhelp/controller/QuestionController.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.emotionalhelp.controller;
+
+import com.epam.emotionalhelp.module.Question;
+import com.epam.emotionalhelp.service.QuestionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/questions")
```

Comment:
```suggestion
@RequestMapping("/v1/questions")
```
Let's also add versioning for REST API.